packages/appsync-modelgen-plugin/src/visitors/appsync-json-metadata-visitor.ts

```diff
@@ -150,6 +152,7 @@ export class AppSyncJSONVisitor<
       models: {},
       enums: {},
       nonModels: {},
+      codegenVersion: this._parsedConfig.codegenVersion || 'unknown',
```

Comment:
Do you have the `unknown` case accounted for in the JS library code?

Reply:
Yes, I have it throw an error that it cannot determine the version.

Reply:
Also we need to make sure this new field not breaking the studio as the studio is in a current transient from `schema.js` to model introspection schema

Reply:
I've talked to Chris before and he said there shouldn't be an issue, but I'll ask him to take a look at this PR.